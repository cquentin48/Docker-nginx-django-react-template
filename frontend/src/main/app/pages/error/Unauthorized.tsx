import { Typography } from "@mui/material";
import React from "react"

export default function Unauthorized (): JSX.Element {
    return (
        <div>
            <Typography variant="h1">
                Accès refusé
            </Typography>
            Une authentification est nécessaire pour se connecter!
        </div>
    );
}
