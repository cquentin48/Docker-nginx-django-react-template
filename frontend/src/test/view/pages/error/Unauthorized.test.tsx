import React from "react";

import renderer from "react-test-renderer";
import Unauthorized from "../../../../main/app/view/pages/error/unauthorized";

test("Is login button correctly rendered", () => {
    // Acts
    const tree: renderer.ReactTestRendererJSON | renderer.ReactTestRendererJSON[] | null = renderer.create(
        <Unauthorized/>
    ).toJSON();

    // Asserts
    expect(tree).toMatchSnapshot()
})
