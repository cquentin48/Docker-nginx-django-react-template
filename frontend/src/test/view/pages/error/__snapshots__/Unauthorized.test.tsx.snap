// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Is login button correctly rendered 1`] = `
<div>
  <h1
    className="MuiTypography-root MuiTypography-h1 css-o2w69a-MuiTypography-root"
  >
    Accès refusé
  </h1>
  Une authentification est nécessaire pour se connecter!
</div>
`;
